src/ui/popup/theme/controls/background-color.tsx

```diff
@@ -9,6 +9,7 @@ interface BgColorEditorProps {
     onChange: (value: BgColorValue) => void;
     canReset: boolean;
     onReset: () => void;
+    cssValue: string;
```

Comment:
Where are we still using cssValue for? Currently it's not being modified or used.

Reply:
It only get used in the color-picker index.tsx if the entered hex color or cssvariable are invalid

Reply:
If the entered hex color or cssvariable is incorrect you don't have to call `props.onChange(...)`.



> Could you please also give some examples of the tests required

Well, if you define a define a variable under `:root { --ex-variable: red }` and then specify this variable in the settings to be used as background-color variable. Will the code be able to convert the variable into a value and then change the colors accordingly.